model_compression_toolkit/core/keras/hessian/hessian_calculator_keras.py

```diff
@@ -0,0 +1,25 @@
+from typing import Dict, List
+
+from model_compression_toolkit.core.common import Graph
+from model_compression_toolkit.core.common.hessian.hessian_calculator import HessianCalculator
+import tensorflow as tf
+
+from model_compression_toolkit.core.common.hessian.hessian_config import HessianConfig
+
+class HessianCalculatorKeras(HessianCalculator):
```

Comment:
It will probably be used with methods related to both once the weights part is implemented. I can remove it for now, but it's probably something that we will need in my next PR.